actions/generate-schedule/users/usersource.go

```diff
@@ -0,0 +1,65 @@
+package users
+
+import (
+	"container/list"
+	"fmt"
+	"sort"
+)
+
+// Source represents a source of usernames for rotation
+type Source interface {
+	//GetUsers() ([]string, error)
```

Comment:
Is this meant to be removed?

Reply:
Whoops... should have been removed. Good catch.